dbt_subprojects/nft/models/nft_metrics/ethereum/nft_ethereum_wallet_metrics.sql

```diff
@@ -138,6 +138,7 @@ reservoir_floors as (
     from {{source('reservoir', 'collection_floor_ask_events')}}
     where 1 = 1
     and valid_until_dt > current_date
+    and valid_until < 100000000000 -- overflow protection
```

Comment:
Maybe we can have one unified number/const threshold for usd value in spellbook in future.

Reply:
the `valid_until` column here is not usd, but a timestamp number